composer.json

```diff
@@ -31,6 +31,7 @@
         "zendframework/zend-serializer": "~2.4.6",
         "zendframework/zend-log": "~2.4.6",
         "zendframework/zend-http": "~2.4.6",
+        "zendframework/zend-captcha": "~2.4.6",
         "magento/zendframework1": "~1.12.16",
```

Comment:
now my big question @okorshenko is who do we have to bribe/blackmail to get them to remove Captcha from this repo and create a new tag to be used in Mage2? Something like 2.0.0 would be nice here as this will fit well with http://semver.org/ as removing Captcha from zendframework1 would mean that this could not be used in Magento1.